x-pack/plugins/maps/public/actions/map_actions.ts

```diff
@@ -295,7 +289,10 @@ export function setQuery({
       searchSessionMapBuffer: getSearchSessionMapBuffer(getState()),
     };
 
-    if (_.isEqual(nextQueryContext, prevQueryContext)) {
+    if (
```

Comment:
Something is not right about this check. nextQueryContext has property forceRefreshTriggeredFromGlobalQueryTime which never gets set on prevQueryContext. forceRefresh, which is added to prevQueryContext is never set on nextQueryContext

Reply:
this went wrong after renaming